coderd/database/migrations/000302_prebuilds.up.sql

```diff
@@ -0,0 +1,76 @@
+-- TODO: using "none" for login type produced this error: 'unsafe use of new value "none" of enum type login_type' -> not sure why
+INSERT INTO users (id, email, username, name, created_at, updated_at, status, rbac_roles, hashed_password, is_system)
+VALUES ('c42fdf75-3097-471c-8c33-fb52454d81c0', 'prebuilds@system', 'prebuilds', 'Prebuilds Owner', now(), now(),
+		'active', '{}', 'none', true);
+
+-- TODO: do we *want* to use the default org here? how do we handle multi-org?
+WITH default_org AS (SELECT id
+					 FROM organizations
+					 WHERE is_default = true
+					 LIMIT 1)
+INSERT
+INTO organization_members (organization_id, user_id, created_at, updated_at)
+SELECT default_org.id,
+	   'c42fdf75-3097-471c-8c33-fb52454d81c0', -- The system user responsible for prebuilds.
+	   NOW(),
+	   NOW()
+FROM default_org;
+
+CREATE VIEW workspace_latest_build AS
```

Comment:
Does `template_versions` have any effect in this query? Can we remove `JOIN template_versions`, etc...?

Reply:
Hhmm, the `template_versions` bit might be extraneous, yes, good catch. I'll look into this tomorrow.
I'm not so sure about your simplification though, but I'll check that too 👍

Reply:
I checked it locally, seems it's working, but ofc we need good unit-test coverage.
The idea is:
- DISTINCT ON (workspace_id) → Keeps only one row per unique `workspace_id`.
- ORDER BY workspace_id, build_number DESC → Sorts row by `build_number` partitioned by `workspace_id`
- The first row is returned (in our case first row is the row with highest `build_number`).

Reply:
My proposal based on assumption that if we have multiple `workspace_builds` with same `workspace_id` - they all should point to the same `template_id`?
Basically meaning that `GROUP BY tv.template_id, wbmax.workspace_id` is equivalent to `GROUP BY wbmax.workspace_id`.

In another words, over time:
- `template version` for workspace CAN change
- `template` CANNOT change
?